Module.yaml

```diff
@@ -0,0 +1,125 @@
+version: 1.0.0
+name: OPCUASubscribe
+description: OPC UA Subscribe connects to OPC UA Server and listens for incoming data messages and forwards the payload to next module.
```

Comment:
If this module subscribes to a server, why would it be of type `process`? If you build a pipeline with this module, do you MUST use another module before this one? If not, then it should be `input` module

Reply:
module acts as standalone, listens for events and forms a JSON and sends it over to next module, something similar like scheduler.